website/static/css/front-page.css

```diff
@@ -0,0 +1,545 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
```

Comment:
Yes the designers are overriding header rules for some headers in the front page.